src/media/js/site/actions/ui.js

```diff
@@ -0,0 +1,11 @@
+import {createAction} from 'redux-actions';
```

Comment:
its goal is to keep application state in one tree, outside the components.

but this isn't application state, it's definitely fitting to keep it part of the component. nothing else needs to know about the dropdown toggle